app/actions/remote/channel.ts

```diff
@@ -131,6 +132,7 @@ export const createChannel = async (serverUrl: string, displayName: string, purp
             type,
         } as Channel;
 
+        ephemeral_store.creatingChannel = true;
```

Comment:
We are setting the ephemeral store value of creating a channel in several places where we create a channel (createChannel, createDirectMessage, createGroupMessage). There is a tradeoff here where for some ms, we are ignoring any websocket event of certain types (channelCreated, groupAdded, dmAdded), but hopefully the window is small enough.

If we don't want to make that tradeoff, we may have to separate (or make the creatingChannel variable accept more different types) to accommodate the three different types of channels.